ballerina/records.bal

```diff
@@ -41,845 +41,752 @@ public type AwsCredentials record {
-#                                 all of your tables in this Region
-# + AccountMaxWriteCapacityUnits - The maximum total write capacity units that your account allows you to provision
-#                                  across all of your tables in this Region
-# + TableMaxReadCapacityUnits - The maximum read capacity units that your account allows you to provision for a new
-#                               table that you are creating in this Region, including the read capacity units
-#                               provisioned for its global secondary indexes (GSIs).
-# + TableMaxWriteCapacityUnits - The maximum write capacity units that your account allows you to provision for a new
-#                                table that you are creating in this Region, including the write capacity units
-#                                provisioned for its global secondary  indexes (GSIs).
-public type LimitDescribtion record {
-    int? AccountMaxReadCapacityUnits?;
-    int? AccountMaxWriteCapacityUnits?;
-    int? TableMaxReadCapacityUnits?;
-    int? TableMaxWriteCapacityUnits?;
-};
-
-# Represents the response after `BatchWriteItem` operation.
-#
-# + ConsumedCapacity - The capacity units consumed by the entire `BatchWriteItem` operation
-# + ItemCollectionMetrics -A list of tables that were processed by `BatchWriteItem` and, for each table, information about
-#                          any item collections that were aﬀected by individual `DeleteItem` or  `PutItem` operations.
-# + UnprocessedItems - A map of tables and requests against those tables that were not processed. The `UnprocessedItems`
-#                      value is in the same form as `RequestItems`, so you can provide this value directly to a subsequent
-#                      `BatchGetItem` operation.
-public type ItemsBatchWriteResponse record {
-    ConsumedCapacity[]? ConsumedCapacity?;
-    map<ItemCollectionMetrics[]>? ItemCollectionMetrics?;
-    map<WriteRequest[]>? UnprocessedItems?;
+public type LimitDescription record {
+    # The maximum total read capacity units that your account allows you to provision across all of your tables in this Region
+    int? accountMaxReadCapacityUnits?;
+    # The maximum total write capacity units that your account allows you to provision across all of your tables in this Region
+    int? accountMaxWriteCapacityUnits?;
+    # The maximum read capacity units that your account allows you to provision for a new table that you are creating in this Region, including the read capacity units
+    int? tableMaxReadCapacityUnits?;
+    # The maximum write capacity units that your account allows you to provision for a new table that you are creating in this Region, including the write capacity units
+    # provisioned for its global secondary  indexes (GSIs).
+    int? tableMaxWriteCapacityUnits?;
+};
+
+# Represents the response after `WriteBatchItem` operation.
+public type WriteBatchItemOutput record {
```

Comment:
Shall we rename the record, How about `BatchItemPostOutput`. is this for creating a new item or updating an item?

Reply:
just trying to avoid starting with a verb. is this name got from dynamodb API?